Sources/Build/PackageVersion.swift

```diff
@@ -60,13 +56,8 @@ func generateData(_ packages: [Package]) -> [String : String] {
 func versionData(package: Package) -> String {
     var data = "public let url: String = \"\(package.url)\"\n"
     data += "public let version: (major: Int, minor: Int, patch: Int, prereleaseIdentifiers: [String], buildMetadata: String?) = "
-    if let version = package.version {
-        data += "\(version.major, version.minor, version.patch, version.prereleaseIdentifiers, version.buildMetadataIdentifier)\n"
-        data += "public let versionString: String = \"\(version)\"\n"
-    } else {
-        data += "(0, 0, 0, [], nil) \n"
-        data += "public let versionString: String = \"0.0.0\"\n"
-    }
+    data += "\(package.version.major, package.version.minor, package.version.patch, package.version.prereleaseIdentifiers, package.version.buildMetadataIdentifier)\n"
+    data += "public let versionString: String = \"\(package.version)\"\n"
```

Comment:
Is this commit tied to the --update support? It seems like it isn't (or almost isn't) and could land independently?